source/vstd/multiset.rs

```diff
@@ -701,3 +701,70 @@ pub use assert_multisets_equal_internal;
 pub use assert_multisets_equal;
 
 } // verus!
+verus! {
+
+impl<A> Multiset<A> {
```

Comment:
Hi! Thanks for the PR!

Why did you place these in a separate `verus!` macro invocation and in a separate `impl` block for `Multiset`?
They either belong in the main `impl` block, or - more likely, to keep with the convention we have for `seq` and `set` - they should probably go in a new file:  `multiset_lib.rs`.

Reply:
Thanks! I have moved the code to `multiset_lib.rs`.